scripts/bundle-dev.config.js

```diff
@@ -0,0 +1,14 @@
+const config = module.require('./bundle.config');
```

Comment:
Use an --env.dev flag instead of creating a new config file?

Then the submodule scripts can look like:

```json
"build-bundle": "webpack --config ../../scripts/bundle.config.js",
"prepublishOnly": "npm run build-bundle && npm run build-bundle --env.dev"
```

Reply:
Done. FWIW, to pass the arguments on to webpack an extra `--` is needed

`npm run build-bundle -- --env.dev"